code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -65,163 +86,339 @@
 
 	lastsinPerson = world.time
 	var/list/possible_titles = list("Pope","Bishop","Lord","Cardinal","Deacon","Pontiff")
-	var/chosen = "[pick(possible_titles)] of Sin"
+	var/chosen = "Hades, [pick(possible_titles)] of Sin"
 	name = chosen
 	real_name = chosen
 
-	world << "<span class='warning'><font size=6>A [name] has entered your reality. Kneel before them.</font></span>"
+	world << "<span class='warning'><font size=4>[name] has entered your reality. Kneel before them.</font></span>"
 	world << 'sound/effects/pope_entry.ogg'
 
-	Appear()
+	Appear(get_turf(src))
+
+/mob/living/simple_animal/hostile/hades/handle_environment(datum/gas_mixture/environment)
+	//space popes are from space, they need not your fickle "oxygen"
+	return
+
+/mob/living/simple_animal/hostile/hades/handle_temperature_damage()
+	//space popes are from space, they don't uh.. something fire burny death
+	return
+
+/mob/living/simple_animal/hostile/hades/death(gibbed)
+	if(!isDoingDeath)
+		src.visible_message("<span class='warning'><font size=4>[src] begins to twist and distort, before snapping backwards with a sickening crunch.</font></span>")
+		spawn(20)
+			src.visible_message("<span class='warning'><font size=4>[src] is being sucked back to their own realm, destabilizing the fabric of time and space itself!</font></span>")
+		playsound(get_turf(src), 'sound/effects/hyperspace_begin.ogg', 100, 1)
+		isDoingDeath = TRUE
+		AIStatus = AI_OFF
+		SpinAnimation()
+		for(var/i in 1 to 5)
+			spawn(i*10)
+				for(var/turf/T in oview(i,src) - oview((i)-1,src))
+					sinShed(T)
+		spawn(60)

```

Comment:
If there's an easier way to do staged things, like a timer proc or something I'll switch it over to that but I'm unaware of such a thing.